xarray/core/dataarray.py

```diff
@@ -2152,6 +2153,10 @@ def interpolate_na(
                   * x        (x) int64 0 1 2 3 4 5 6 7 8
 
             The gap lengths are 3-0 = 3; 6-3 = 3; and 8-6 = 2 respectively
+        keep_attrs : bool, optional
```

Comment:
This should say something about the global option, but we probably need to make that change in a lot of docstrings, so lets do that later. This should mention the default though

Reply:
It does mention the default just below, but yeah you're right it should be in this line.